ddpui/asgi.py

```
@@ -8,10 +8,54 @@
+import traceback
+from datetime import datetime
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ddpui.settings")
 
+tracemalloc.start(25)
+
+
+def _dump_memory_profile(signum, frame):
+    """Send SIGUSR1 to dump memory and thread info: kill -SIGUSR1 <pid>"""
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    dump_path = f"/tmp/memory_dump_{timestamp}.txt"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/memory_dump_"

(S108)